enos/enos.py

```diff
@@ -359,18 +411,31 @@ def cartesian(d):
                     run_ansible([playbook_path], inventory_path, env['config'])
 
 @enostask("""
-usage: enos backup [--backup_dir=BACKUP_DIR ] [-vv|-s|--silent]
+usage: enos backup [--backup_dir=BACKUP_DIR  [-e ENV|--env=ENV]
+                   [-v|-vv|-s|--silent]
 
 Backup the environment
 
 Options:
-  --backup_dir=BACKUP_DIR   Backup directory.
-  -h --help                 Show this help message.
-""")
+  --backup_dir=BACKUP_DIR  Backup directory.
+  -e ENV --env=ENV         Path to the environment yaml file. You should
+                           use this option when you wanna link a specific
+                           experiment [default: %s].
+  -h --help            Show this help message.
+  -v --version         Show version number.
+  -vv                  Verbose mode.
+  -s --silent          Quiet mode.
+""" % os.path.join(SYMLINK_NAME, 'env'))
 def backup(env = None, **kwargs):
-    backup_dir = kwargs['--backup_dir']
-    if backup_dir is None:
-        backup_dir = SYMLINK_NAME
+
+    # Override default result directory name when --env parameter is given
```

Comment:
This piece of code seems duplicated 4 times. Is it possible to factorize it somehow ?